sources/api/netdog/test_data/wicked/eno1-ra.xml

```diff
@@ -0,0 +1 @@
+<interface><name>eno1</name><control><mode>boot</mode><link-detection><require-link></require-link></link-detection></control><ipv4:dhcp><enabled>true</enabled></ipv4:dhcp><ipv6><accept-ra>router</accept-ra></ipv6></interface>
```

Comment:
Is it correct to do this if the interface is not configured for IPv6 otherwise?

Reply:
I think from further looking, this probably won't affect anything so I'm fine with it remaining even if it looks a little odd.

Reply:
This feels pretty safe given that it only gets added if the interface is configured via kernel command line, which currently only happens in AWS/VMware variants where DHCP6 is optional by default.  It would be a bit of a chore to specify via kernel command line with other variants.

Reply:
Ideally, and I mentioned this in my comments with @markusboehme , we will make this more configurable in the future via net config.